ios/tools/sourcery/templates/Shared.stencil

```diff
@@ -4,6 +4,7 @@ import Combine
 
 {% for type in types.protocols where type.name | hasSuffix:"UseCase" %}
   {% for method in type.methods where method.name | hasPrefix:"callAsNative(" %}
+// sourcery:begin: AutoMockable
```

Comment:
Please fix all changes in one PR #67. Or maybe separate a chore for setting up a new template/stencil for this one

Reply:
@suho updated at [b5c86d3](https://github.com/markgravity/kmm-ic/pull/67/commits/b5c86d34cf43b9e1cac1bdf1dc2ae0f36ddf159a)